aiidalab_widgets_base/structures.py

```diff
@@ -653,25 +659,28 @@ def __init__(self, title=""):
 
         super().__init__([self.smiles, self.create_structure_btn, self.output])
 
-    def make_ase(self, species, positions):
+    def make_ase(self, species, positions, smiles):
```

Comment:
We are changing the API of several methods, which is not so good, in principle. But, they should not be called outside of this class, so I would suggest making them "internal":

```suggestion
    def _make_ase(self, species, positions, smiles):
```